modules/inputSpinner/InputSpinner.tsx

```diff
@@ -0,0 +1,157 @@
+import React, { useCallback, useEffect, useMemo, useState } from "react";
+import {
+  ThemeStyleSheetFactory,
+  StandardProps,
+  WithStylesProps
+} from "@diana-ui/types";
+import { withStyles } from "@diana-ui/base";
+import { ButtonText } from "@diana-ui/typography";
+
+// @ts-ignore
+export interface IProps extends StandardProps<"input"> {
+  min?: number;
+  max?: number;
+  step?: number;
+  value?: number;
```

Comment:
Should we have an `initialValue` prop?

Reply:
the value can be passed and act as initialValue, but it's true that if we do it, it starts to be controlled and needs the onChange to change value. It might be a good addition